ACE/ace/Functor.h

```diff
@@ -200,7 +200,22 @@ class ACE_Export ACE_Hash<unsigned long>
   unsigned long operator () (unsigned long t) const;
 };
 
-#if (ACE_SIZEOF_LONG < 8)
+template<>
+class ACE_Export ACE_Hash<long long>
+{
+public:
+  /// Simply returns t
+  unsigned long operator () (long long t) const;
+};
+
+template<>
+class ACE_Export ACE_Hash<unsigned long long>
+{
+public:
+  unsigned long operator () (unsigned long long t) const;
+};
+
+#if ((ACE_SIZEOF_LONG < 8) && (ACE_SIZEOF_LONG_LONG < 8))
```

Comment:
Are there platforms where long long is smaller 64bit?

Reply:
I fixed this in a subsequent commit.